x-pack/test/api_integration/apis/monitoring/logs/multiple_clusters.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import multipleClustersFixture from './fixtures/multiple_clusters';
+
+export default function ({ getService }) {
+  const supertest = getService('supertest');
+  const esArchiver = getService('esArchiver');
+
+  describe('multiple cluster', () => {
+    const archive = 'monitoring/logs_multiple_clusters';
+    const timeRange = {
+      min: '2019-08-23T14:14:31.686Z',
+      max: '2019-08-23T15:14:31.686Z'
+    };
+    const codePaths = ['logs'];
+
+    before('load archive', () => {
+      return esArchiver.load(archive);
+    });
+
+    after('unload archive', () => {
+      return esArchiver.unload(archive);
+    });
+
+    it('should get log types at the cluster level', async () => {
```

Comment:
Some additional documentation might be good for this file, just so it's a bit more explicit about what's being tested. It _seems_ like this is posting to the endpoint for the cluster without logs and then ensuring that we return a response that at least contains a key for `types` even though the corresponding value is an empty array, which makes sense since there's no logs. If that's right, it might be good to make that more explicit in this file, since it's hard to infer that's the case in the archived data that's being loaded.

Additionally, should we add testing for logs flowing in for multiple clusters? (Not in this PR, but perhaps as a thought for the future. If so, we'd probably want to change the naming of the fixture and the archive in this PR?)